jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -1477,6 +1477,27 @@ private char compareValue(byte b) {
 		return Character.toLowerCase(Character.toUpperCase(helpers.byteToCharUnsigned(b)));
 	}
 
+	private boolean charValuesEqual(char c1, char c2) {
```

Comment:
This method should also be `static` as it does not use any fields.

Reply:
This method is missing in Java 8 pConfig.

Reply:
Added this method to the Java 8 section of String; made `compareValue(int/char/byte)` in Java 9+ section `static` as well, since they do not use any non-static fields (the Java 8 versions are already static).